scripts/update-apk.sh

```diff
@@ -17,34 +20,49 @@ fi
 git clone --quiet --branch=apk https://fossasia:$GITHUB_API_KEY@github.com/fossasia/open-event-attendee-android apk > /dev/null
 cd apk
 
-if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
-	/bin/rm -f  *
-else
-	/bin/rm -f open-event-attendee-dev-*
+if ! $PR_FOR_RELEASE; then
+    if $IS_DEV_BRANCH; then
+        /bin/rm -f open-event-attendee-dev-*
+    else
+        /bin/rm -f  *
+    fi
 fi
 
 find ../app/build/outputs -type f -name '*.apk' -exec cp -v {} . \;
 find ../app/build/outputs -type f -name '*.aab' -exec cp -v {} . \;
 
 
 for file in app*; do
-    if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
+    if $IS_DEV_BRANCH; then
         if [[ ${file} =~ ".aab" ]]; then
-            mv $file eventyay-attendee-master-${file}
+                mv $file eventyay-attendee-dev-${file}
         else
-            mv $file eventyay-attendee-master-${file:4}
+                mv $file eventyay-attendee-dev-${file:4}
         fi
 
-    elif [ "$TRAVIS_BRANCH" == "$DEPLOY_BRANCH" ]; then
+    else
         if [[ ${file} =~ ".aab" ]]; then
-                mv $file eventyay-attendee-dev-${file}
+            mv $file eventyay-attendee-master-${file}
         else
-                mv $file eventyay-attendee-dev-${file:4}
+            mv $file eventyay-attendee-master-${file:4}
         fi
 
     fi
 done
 
+if $IS_PUBLISH_BRANCH ;then
+    cd ..
+    gem install fastlane
+    if $PR_FOR_RELEASE ;then
+        echo "cool, so you're trying to release a new version. Lets check things out first, hold on..."
+        fastlane supply --aab ./apk/eventyay-attendee-master-app-playStore-release.aab --skip_upload_apk true --track alpha --json_key ./scripts/fastlane.json --package_name $PACKAGE_NAME $FASTLANE_DRY_RUN
+        exit 0
+    else
+        echo "Hurray....! Lets release a new build for our app"
+        fastlane supply --aab ./apk/eventyay-attendee-master-app-playStore-release.aab --skip_upload_apk true --track alpha --json_key ./scripts/fastlane.json --package_name $PACKAGE_NAME
```

Comment:
Don't duplicate. That's why `$FASTLANE_DRY_RUN` should be assigned conditionally, not this statement

Reply:
@iamareebjamal have a look